docs/src/user_guide.ipynb

```diff
@@ -137,106 +147,274 @@
+    "\n",
+    "A Temporian program is a graph of `tp.EventSetNodes` connecting operators. A graph is executed with the function `tp.run`. It signature is `tp.run(<outputs>, <inputs>)`.\n",
     "\n",
     "The `<outputs>` can be specified as an `EventSetNode`, a list of `EventSetNodes`, or a dictionary of names to `EventSetNodes`, and the result of `tp.run()` will be of the same type. For example, if `<outputs>` is a list of three `EventSetNodes`, the result will be a list of the three corresponding `EventSets`.\n",
     "\n",
-    "The `<inputs>` can be specified as a dictionary of `EventSetNodes` to `EventSets`, a dictionary of names to `EventSets`, a list of `EventSets` or a single `EventSet`, which lets Temporian know the `EventSetNodes` of the graph that each input `EventSet` corresponds to. In the case of a dictionary of names to `EventSets`, the names must match the names of `EventSetNodes` in the graph, and in the case of a list or single `EventSet`, the names of those `EventSets` must do the same. If we specify the inputs as a dictionary, we could skip passing a name to `a_evset`.\n",
+    "The `<inputs>` can be specified as a dictionary of `EventSetNodes` to `EventSets`, a dictionary of names to `EventSets`, a list of `EventSets` or a single `EventSet`, which lets Temporian know the `EventSetNodes` of the graph that each input `EventSet` corresponds to. In the case of a dictionary of names to `EventSets`, the names must match the names of `EventSetNodes` in the graph, and in the case of a list or single `EventSet`, the names of those `EventSets` must do the same. If we specify the inputs as a dictionary, we could skip passing a name to `a_evset`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd4142a0-df47-4ca3-aafe-7a105b4f2deb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_node = tp.input_node([(\"value\", tp.float64)])\n",
+    "result_1_node = tp.simple_moving_average(input_node, window_length=0.5)\n",
+    "result_2_node = tp.simple_moving_average(input_node, window_length=1.0)\n",
+    "result_3_node = result_1_node - result_2_node\n",
     "\n",
+    "result = tp.run([result_1_node,result_2_node, result_3_node], {input_node: evset})\n",
+    "    \n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32dd49a8-8847-4df1-be17-b844b92f4211",
+   "metadata": {},
+   "source": [
     "**Remarks:**\n",
     "\n",
-    "- It's important to distinguish between _`EventSets`_, such as `a_evset`, that contain data, and _`EventSetNodes`_, like `a_node` and `b_node`, that connect operators together and compose the computation graph, but do not contain data.\n",
+    "- It's important to distinguish between a `tp.EventSet`, such as `evset`, that contain data, and a `tp.EventSetNode`, like `input_node`, that connect operators together and compose the computation graph, but do not contain data.\n",
```

Comment:
It's important to distinguish between a `tp.EventSet`, such as `evset`, that contains data, and a `tp.EventSetNode`, like `input_node`, that connect operators together and compose the computation graph, but do not contain data.